src/modules/APM.ts

```diff
@@ -63,6 +68,51 @@ export const startExecutionTrace = async (name: string): Promise<Trace> => {
   return new Trace(id, name);
 };
 
+/**
+ * Starts an AppFlow with the specified name.
+ *
+ * On starting two flows with the same name, the older flow will end with a force abandon end reason.
+ * The AppFlow name cannot exceed 150 characters; otherwise, it's truncated,
+ * leading and trailing whitespaces are also ignored.
+ *
+ * @param name - The name of the AppFlow. It cannot be an empty string or null.
+ *               A new AppFlow is started if APM is enabled, the feature is enabled,
+ *               and the Instabug SDK is initialized.
+ */
+export const startFlow = (name: string) => {
+  NativeAPM.startFlow(name);
+};
+
+/**
+ * Ends an AppFlow with the given name.
+ *
+ * @param name - The name of the AppFlow to end. It cannot be an empty string or null.
+ */
+export const endFlow = (name: string) => {
+  NativeAPM.endFlow(name);
+};
+
+/**
+ * Sets custom attributes for an AppFlow with a given name.
+ *
+ * Setting an attribute value to null will remove the corresponding key if it already exists.
+ * Attribute keys cannot exceed 30 characters and leading/trailing whitespaces are ignored.
+ * Empty strings or null for attribute keys are not accepted.
+ *
+ * Attribute values cannot exceed 60 characters and leading/trailing whitespaces are ignored.
+ * Empty strings for attribute values are not accepted, but null can be used to remove an attribute.
+ *
+ * If an AppFlow is ended, attributes will not be added and existing ones will not be updated.
+ *
+ * @param name - The name of the AppFlow. It cannot be an empty string or null.
+ * @param key - The key of the attribute. It cannot be an empty string or null.
+ * @param [value] - The value of the attribute. It cannot be an empty string. Use null to remove the attribute.
+ */
+
+export const setFlowAttribute = (name: string, key: string, value?: string) => {
```

Comment:
Do you think we should also allow setting the value to `null` besides `undefined`?

```suggestion
export const setFlowAttribute = (name: string, key: string, value?: string | null) => {
```